components/identity.oauth.uma.resource.endpoint/src/main/java/org/wso2/carbon/identity/oauth/uma/resource/endpoint/impl/ResourceRegistrationApiServiceImpl.java

```diff
@@ -1,283 +0,0 @@
-/*
```

Comment:
Why this whole class is deleted?

Reply:
It's not deleted. Due to the renaming of the full package, it's giving like this.